packages/lib/src/vis/tiles/api.ts

```diff
@@ -0,0 +1,47 @@
+import type { NdArray } from 'ndarray';
+import type { Vector2 } from 'three';
+
+import type { TextureTypedArray } from '../heatmap/models';
+import type { Size } from '../models';
+
+export function getLevelsSize(
+  size: Size,
+  tileSize: Size,
+  evenLevelSize = false
+): Size[] {
+  const levelsSize = [] as Size[];
```

Comment:
```suggestion
  const levelsSize: Size[] = [];
```

(replace cast by type declaration)